components/src/core/components/CardTable/Cell/default.scss

```diff
@@ -12,20 +12,4 @@
       cursor: unset !important;
     }
   }
-  a {
-    color: red;
```

Comment:
What was this red color related to? Was it preserved after refactoring?

Reply:
@gayan-orangehrm can you check and confirm it's not used

Reply:
@Super-Chama confirmed. this red colour is not in used